addon/templates/components/common-dashboard.hbs

```diff
@@ -19,7 +19,17 @@
     changeSchool=changeSchool
     courseFilters=courseFilters
     toggleCourseFilters=toggleCourseFilters
-  }}
+    selectedCohorts=@selectedCohorts
+    selectedCourseLevels=@selectedCourseLevels
+    selectedCourses=@selectedCourses
+    selectedSessionTypes=@selectedSessionTypes
+    selectedTerms=@selectedTerms
+    onClearFilters=(action @onClearFilters)
```

Comment:
Since it already expects an action shouldn't this be `onClearFilters=@onClearFilters`? Is the action helper needed? Same for the next lines as well.

Reply:
Fixed, amended, and pushed.